src/simulation/Simulation.h

```diff
@@ -206,9 +206,7 @@ class Simulation
 	static int remainder_p(int x, int y);
 	static float remainder_p(float x, float y);
 
-	String ElementResolve(int type, int ctype) const;
-	String BasicParticleInfo(Particle const &sample_part) const;
-
+	String ElementResolve(int type, int ctype, int tmp = 0, int tmp4 = 0, bool includeCtype = true) const;
```

Comment:
I haven't looked into this, does this optional parameter really need to be optional?